xbanish.1

```diff
@@ -9,7 +9,7 @@
 .Op Fl a
 .Op Fl d
 .Op Fl i Ar modifier
-.Op Fl m Oo Ar w Oc Ns Ar nw|ne|sw|se
+.Op Fl m Oo Ar w Oc Ns Ar nw|ne|sw|se|+/\-xy
```

Comment:
Not sure what's the proper way to indicate it accepts X Geometry values.

Reply:
```groff
.It Fl m Oo Ar w Oc Ns Ar nw|ne|sw|se|\(+-x\(+-y
```

This works well on my system. Not familiar enough with troff to know weather it's portable or not.